app/components/edit/contributor_component.rb

```diff
@@ -36,11 +36,11 @@ def contributors_data
     end
 
     def role_label
-      'Role'
+      t('works.edit.fields.contributors.role.label')
     end
 
     def organization_label
-      'Organization name'
+      t('works.edit.fields.contributors.organization_name.label')
```

Comment:
refactor: while editing the component and locale file, decided to just move these labels into the locale file